### ISSUE TITLE
Ubuntu 17.10 no longer working with omf

### ISSUE THREAD

**Issue by DWinterfell** [oh-my-fish/oh-my-fish#616]
Hey guys, ever since Ubuntu did a version update, `omf` no longer works.  It initially installs correctly, but after a logout or a reboot, it quits working. 👎 
Even after an uninstall and reinstall of `omf` does not stay installed.  What's going on guys?

**Comment by MrGKanev** [comment]
Yep, the same problem on Ubuntu 18.04. Tried reinstalling and everything goes smoothly until I try to make it the default. Then everything goes crazy. These are the problems every time it displays:
1. Doesn't have permissions - I have it 777 just to see what will happen. It still says no permissions
2. Can't find fish
It is so silly that I'm not sure if it gives me these errors on random or there is really a problem there. I tried everything from the FAQ page of this git.

**Comment by oranja** [comment]
Hi @DWinterfell and @MrGKanev, thanks for reporting this issue.
If this problem still persists, please help us understand the problem a little more and we'd love to help.
* Which version of _fish_ is installed after the update you mentioned? Do you know which version was installed before that?
* Where are you getting your _fish_ packages from? Is it the official Ubuntu repositories or the fish-shell PPA?
* Is it just _omf_ that stopped working? With _fish_ but without _omf_ is everything working as usual?
* Can you please paste here some examples of errors that you see on screen, with a short description of what you were doing just before those errors appeared (AKA how to reproduce the errors?)

**Comment by MrGKanev** [comment]
Hi @oranja,
Yes, the problem is still here at least for me but after reinstalling it 2-3 times now only one problem stands. Here is the info you wanted:
- Version: fish, version 2.7.1
- I installed fish from Ubuntu repositories then installed oh-my-fish. 
- I think the combination of the two stopped working. I have not tried installing only fish. OMF as a command doesn't work until I start fish. (Not sure if that is my bad)
- The only error i get more then 10 times at a time: <E> fish: Unable to open universal variable file '/home/mrgkanev/.config/fish/fishd.Monster': Permission denied . Somewhere in the middle it says : "Welcome to fish, the friendly interactive shell" and another 3-4 errors like the ones before.
Still not sure if it is user error (mine) or it is a problem. I am working with omf on a mac and there i haven't had problems like this.
